### ISSUE TITLE
Is there any Go implementation?

### ISSUE THREAD

**Issue by vendersen** [0xBEEFCAF3/munstr#42]
Is there any Go implementation?

**Comment by arminsabouri** [comment]
Nope! Might be easier to consider a rust implementation instead. secp2561-zkp crate recently added musig2 to support.

**Comment by arminsabouri** [comment]
Not familiar with any musig2 impls in go. But also not super hard to implement it as we do in this repo
